_includes/tutorials/aggregating-minmax/flinksql/code/tutorial-steps/dev/expected-print-output-topic.log

```diff
@@ -0,0 +1,4 @@
+ release_year min_total_sales max_total_sales
+         2019       385082142       856980506
```

Comment:
nit: are there any stock number formatters we could apply? These numbers are hard to look at - I realize that's not the point of the tutorial

Reply:
I agree... other SQL dialects have convenient built-in functions for this but I'm not (yet) finding anything in Flink SQL.  will keep looking